tests/foreman/cli/test_contenthost.py

```diff
@@ -48,53 +46,42 @@ class TestContentHost(CLITestCase):
     LIBRARY = None
     DEFAULT_CV = None
 
-    def setUp(self):
+    # pylint: disable=unexpected-keyword-arg
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Logic is moved to setUpClass() instead of setUp() since it is not required to run for each test